src/publisher/admin.py

```diff
@@ -1,5 +1,5 @@
 from django.contrib import admin
-from . import models
+from . import models, aws_events
```

Comment:
well, you shouldn't strictly need to name your module after the specific technology `aws`, `events` would make a nice pair with `models` (which is not `postgresql_models`)